projects/uitdatabank/docs/entry-api/project-flow.md

```diff
@@ -0,0 +1,60 @@
+
+## Step 2: validation by publiq vzw
+
+As soon as you have finished developing your integration with the Entry API, you send a minimum of 5 of events to our test environment. You contact us via `vragen@publiq.be` and provide us the identifiers of the created test events. We will then validate your integration as soon as possible. 
+
+In this validation process we assess the integration in two manners:
+1. Is the **created data** qualitative, relevant and valuable for public agendas such as www.uitinvlaanderen.be?
+2. Is the **integration made robust**, e.g. is it possible to update and delete content through the integration?
+
+It is important that you go through the checklist below in advance and validate your integration against it. By doing this in advance, you ensure a smooth transition to the production environment!
+
+### Checklist for a succesfull integration
+#### Mandatory fields
+- [ ] **name**: only use plain text 
+- [ ] **eventtype**: use a valid ID from the [UiTdatabank taxonomy](taxonomy.uitdatabank.be/terms)
+- [ ] **location**: UiTdatabank already contains a lot of locations. Use existing locations as much as possible. Therefore, a search on existing locations in UiTdatabank (see [docs](https://docs.publiq.be/docs/uitdatabank/55c6a99f43e7e-place-get)) should be built in in your application. Or alternatively (if all your events take place on the same location for example), the identifier of that location in UiTdatabank should be kept in your application so that each new event is created with the same location ID. Only when a location does not exist yet in UiTdatabank, a new one can be created. Always save the identifiers of the created locations in the application, so that later on you can update or delete the location if necessary
+- [ ] **calendar**: use the provided calendarTypes in a correct way, and use the ISO-8601 standard
+
+> In order to create an UiTPAS event, the fields `organizer` and `priceInfo` are also mandatory!
+
+#### Optional fields
+- [ ] **description**: only use plain text (and line breaks)
+- [ ] **age range**
+- [ ] **mediaObject (image)**: add an inspiring photo for your event, with respect to copyright
+- [ ] **organizer**: UiTdatabank already contains a lot of organizers. Use existing organizers as much as possible. Therefore, a search on existing organizers in UiTdatabank (see [docs](https://docs.publiq.be/docs/uitdatabank/c66fa72affb6e-organizer-get)) should be built in in your application. Or alternatively (if all your events have the same organizer for example), the identifier of that organizer in UiTdatabank should be kept in your application so that each new event is created with the same organizer ID. Only when an organizer does not exist yet in UiTdatabank, a new one can be created. Always save the identifiers of the created organizer in the application, so that later on you can update or delete the organizer if necessary.
```

Comment:
Same issue here as with the link for locations (see https://github.com/cultuurnet/apidocs/pull/82/files#r1004662491)

Reply:
Noticed you removed this as well so resolving this 👍